### ISSUE TITLE
Move mouse to an absolute position.

### ISSUE THREAD

**Issue by hgaiser** [emberian/evdev#142]
I'm not entirely sure what I'm doing is how it is intended, but my goal is to set the mouse cursor to an absolute position on screen (not a relative position). I assumed I could do something like in the [virtual joystick](https://github.com/emberian/evdev/blob/main/examples/virtual_joystick.rs) example, but my cursor does not move with that example.

Should I be creating different events?

Relative mouse movements did work. A workaround could be to compute the relative movement required to move to a certain absolute position, but that's kinda ugly.

**Comment by mattfbacon** [comment]
As far as I know mice always use relative, not absolute events. You can try to emit absolute events anyway, but software won't expect it and probably won't work correctly. For example https://stackoverflow.com/questions/5190921/simulating-absolute-mouse-movements-in-linux-using-uinput

**Comment by coolreader18** [comment]
I don't think this is within the realm of this library, given the above comment.
